src/xGadgetron/pGadgetron/Gadgetron.py

```diff
@@ -41,6 +41,10 @@
 import sirf.pygadgetron as pygadgetron
 import sirf.pysirf as pysirf
 
+import sirf.select_module as select_module
+select_module.module = 'pygadgetron'
+import sirf.parameters as parms
```

Comment:
should these all be `params`?

Reply:
no objections, could even be `parameters` or `set_get_methods` etc etc